src/screens/ProfileScreen/index.js

```diff
@@ -196,7 +196,6 @@ const ProfileScreen = ({route}) => {
   const {refreshCount} = useResetContext();
   const {mappingColorFeed} = useCoreFeed();
   const LIMIT_PROFILE_FEED = 10;
-  const refBottomSheet = React.useRef();
   const TYPE_GALLERY = 'gallery';
 
   const {feeds} = myProfileFeed;
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The removal of `refBottomSheet` suggests that the bottom sheet component might have been replaced or removed entirely. If this is the case, make sure that all instances where `refBottomSheet` was used have been properly refactored to avoid null reference errors. Otherwise, the changes look good.
```diff
- const refBottomSheet = React.useRef();
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->

Reply:
not use